File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/rust/csc32.rs
fn main() {
    for i in 1..101 {
        fizzbuzz(i)
    }
}
fn fizzbuzz(i: i32) {
    if i % 3 == 0 && i % 5 == 0 {
        return println!("fizzbuzz");
    }
    if i % 3 == 0 {
        return println!("fizz");
    }
    if i % 5 == 0 {
        return println!("buzz");
    }
    return println!("{i}");
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/php/csc32.php
<?php 
function fizzbuzz($num){
    if ($num % 3 == 0 && $num % 5 == 0){
       return  "fizzbuzz";
    }

    if ($num % 3 == 0 ){
       return  "fizz";
    }

    if ($num % 5 == 0){
       return  "buzz";
    }
    return $num;
}

for ($i=1; $i <101 ; $i++) { 
   echo fizzbuzz($i) . "\n";
}
?>